README.md

```diff
@@ -15,6 +15,19 @@ This repository contains Cloudify's system tests which in most cases mean that t
 
 In addition, plugins functionality is tested and Cloudify's examples.
 
+* requirements:
+
+pip version 1.5 or higher.
+python-devel (centos).
+python-dev (ubuntu).
+gcc (ubuntu, centos).
+
+yum install python-devel gcc -y
+apt-get install python-dev gcc -y
+
+add server_name to the manager-inputs file.
```

Comment:
I don't understand what I need to do from this and it's not `server_name`, it's `manager_server_name` as far as I know.